src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/ReleaseComObjectTests.Windows.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices.Tests.Common;
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests
+{
+    public partial class ReleaseComObjectTests
+    {
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void ReleaseComObject_ValidComObject_Success()
+        {
+            var comObject = new ComImportObject();
+            Assert.Equal(0, Marshal.ReleaseComObject(comObject));
+        }
```

Comment:
can move this test to ```ReleaseComObjectTests.cs``` and use  ```PlatformSpecific(TestPlatforms.Windows)]```
No need to have an extra file just for 1 tests

Reply:
In this case, I would agree. I am hoping eventually to support this API on non-Windows.

Reply:
Not sure I can do that, because `ComImportObject` is a Windows com object and defined in `CommonTypes.Windows.cs`

Reply:
@hughbe That is fine then.